nix/upm/default.nix

```diff
@@ -3,11 +3,16 @@
   rev,
   makeWrapper,
 }:
-buildGoModule {
+buildGoModule rec {
   pname = "upm";
   version = rev;
 
-  src = ../../.;
+  src = builtins.path {
+    name = "${pname}-${version}-src";
+    path = ../../.;
+    filter = path: type: type != "directory" ||
+      builtins.baseNameOf path != "test-suite";
```

Comment:
i'm thinking of changing this to an allowlist instead of a blocklist...

Reply:
Yeah, can it be src and internal?

Reply:
done in [6290c1b](https://github.com/replit/upm/pull/126/commits/6290c1b419dc3a46c81272ab171ec3741f631def)

Reply:
that didn't actually work, i went back to a blocklist because it's easier and (hopefully) equivalent